decidim-core/app/views/decidim/shared/_resource_actions.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="relative ml-auto">
+  <button id="dropdown-trigger-resource-<%= resource.id %>"
+          data-component="dropdown"
+          data-target="dropdown-menu-resource-<%= resource.id %>"
+          aria-label="<%= t("decidim.resource.controls_label") %>">
+    <span class="sr-only"><%= t("decidim.resource.controls_label") %></span>
+    <%= icon "more-2-fill" %>
+    <%= icon "more-2-fill" %>
```

Comment:
This is intentional, to avoid the css rules from here. 
https://github.com/decidim/decidim/blob/802ee6a2434f8580a87c55a6956748a092cec4c1/decidim-core/app/packs/stylesheets/decidim/_dropdown.scss#L9-L11

Reply:
Done in [6c3a7bc](https://github.com/decidim/decidim/pull/13377/commits/6c3a7bcbfd17addbd2bf7c59b275aa3f33e11c43)

Reply:
Let's add that as a comment, I've left a suggestion, feel free to improve on it